provider/src/main/java/com/cloudera/director/google/sql/GoogleCloudSQLInstanceTemplateConfigurationProperty.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2015 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cloudera.director.google.sql;
+
+import static com.cloudera.director.spi.v1.database.DatabaseServerInstanceTemplate.DatabaseServerInstanceTemplateConfigurationPropertyToken.ADMIN_PASSWORD;
+import static com.cloudera.director.spi.v1.database.DatabaseServerInstanceTemplate.DatabaseServerInstanceTemplateConfigurationPropertyToken.ADMIN_USERNAME;
+import static com.cloudera.director.spi.v1.database.DatabaseServerInstanceTemplate.DatabaseServerInstanceTemplateConfigurationPropertyToken.TYPE;
+
+import com.cloudera.director.spi.v1.model.ConfigurationProperty;
+import com.cloudera.director.spi.v1.model.ConfigurationPropertyToken;
+import com.cloudera.director.spi.v1.model.util.SimpleConfigurationPropertyBuilder;
+
+
+/**
+ * Google Cloud SQL instance template configuration properties.
+ */
+public enum GoogleCloudSQLInstanceTemplateConfigurationProperty implements ConfigurationPropertyToken {
+
+  TIER(new SimpleConfigurationPropertyBuilder()
+      .configKey("tier")
+      .name("Tier")
+      .defaultValue("D1")
+      .defaultDescription("The tier of your database. This affects performance and how much you will be charged.")
+      .widget(ConfigurationProperty.Widget.OPENLIST)
+      .addValidValues(
```

Comment:
Is there an API to query the set of valid instance types for Cloud SQL? Would be nice to query it at runtime (possibly with caching) rather than hard-coding, because this means for any additional instances, we need a new Director release (or at least a plugin release) to update it.

Not necessary to make the change in this PR (would rather this version is submitted as-is first), but please add a TODO and file an issue to track this in the future so that it's not lost or forgotten.